specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2019-11-01/costmanagement.json

```diff
@@ -379,6 +379,54 @@
         }
       }
     },
+    "/{scope}/providers/Microsoft.CostManagement/views/render": {
```

Comment:
This URI needs to specify what view to render.  You can't have POST actions on a resource type collection

Reply:
Removed all the views/render api changes from the PR.
Had a talk with the team after reading all the comments and we will be making changes to the api and redocumenting in a different PR. This PR will only add the two missing fields. Is this good for us to get unblocked on the small changes in this PR?